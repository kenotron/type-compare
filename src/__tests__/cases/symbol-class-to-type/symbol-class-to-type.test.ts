import { testHarness } from "../../test-harness";
testHarness("symbol-class-to-type");