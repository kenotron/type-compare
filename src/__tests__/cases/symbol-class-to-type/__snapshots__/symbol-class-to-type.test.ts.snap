// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test case "symbol-class-to-type" 1`] = `
{
  "messages": [
    "Export "Foo" symbol kind changed from a value symbol to a non-value symbol",
  ],
  "minChangeType": "major",
}
`;
